package.json

```diff
@@ -27,25 +27,24 @@
   },
   "homepage": "https://github.com/NerdWalletOSS/apollo-invalidation-policies#readme",
   "peerDependencies": {
-    "@apollo/client": "^3.3.0"
+    "@apollo/client": "^3.3.7"
```

Comment:
While this is the latest version i don't think we need to emit a warning to folks on like `3.3.6` for example by requiring at least 3.3.7 in the peer. Thoughts?

Reply:
Sure, I agree. I will revert this.

Reply:
reverted